mmselfsup/engine/optimizers/layer_decay_optim_wrapper_constructor.py

```diff
@@ -87,14 +84,19 @@ def add_params(self, params: List[dict], module: nn.Module,
             optimizer_cfg (dict): The configuration of optimizer.
             prefix (str): The prefix of the module.
         """
+        # get param-wise options
+        custom_keys = self.paramwise_cfg.get('custom_keys', {})
```

Comment:
Please note this modification will be a BC breaking. Please check whether the corresponding parts in MAE, CAE, SimMim need to be refined.

Reply:
I checked the configs and modified them ,please review is again